code/game/objects/items/ego_weapons/teth.dm

```diff
@@ -262,3 +262,48 @@
 	attack_verb_continuous = list("slices", "slashes", "stabs")
 	attack_verb_simple = list("slice", "slash", "stab")
 	hitsound = 'sound/weapons/bladeslice.ogg'
+
+/obj/item/ego_weapon/hearth //From my sweet home.
+	name = "Hearth"
+	desc = "Home sweet home. Warmth and safety aplenty."
+	special = "This weapon has a ranged attack."
+	icon_state = "soulmate"
+	force = 18
+	attack_speed = 1.2
+	damtype = BLACK_DAMAGE
+	armortype = BLACK_DAMAGE
+	attack_verb_continuous = list("swipes", "slashes")
+	attack_verb_simple = list("swipe", "slash")
+	hitsound = null
+	attribute_requirements = list(
+							TEMPERANCE_ATTRIBUTE = 20
+							)
+	var/ranged_cooldown
```

Comment:
Instead of all of these variables it can simply have high reach as kirie suggested, unless you have something else in mind now.